sql/core/src/main/scala/org/apache/spark/sql/Dsl.scala

```diff
@@ -105,6 +105,8 @@ object Dsl {
   def countDistinct(expr: Column, exprs: Column*): Column =
     CountDistinct((expr +: exprs).map(_.expr))
 
+  def approxCountDistinct(e: Column, rsd: Double = 0.05): Column =
```

Comment:
we need java support. can you remove the default value and just have two methods?